index.js

```diff
@@ -0,0 +1,17 @@
+const userLogin = document.querySelector("#user-login");
+
+function setNameIfFound(name){
+    if (name) {
+        userLogin.innerHTML = `Hello, ${name}!`;    
+    }
+};
+
+const fetchData = () => {
+    fetch('https://staging-api.realdevsquad.com/users/self')
+    .then(res => res.json())
+    .then(res => {
+        setNameIfFound(res.first_name)
+    });
+};
+
+fetchData();
```

Comment:
While this works because you load your js later in the HTML, it may be worthwhile wrapping this in document loaded callback: https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event

Reply:
Yeah, had not thought of that. Thanks!